src/main/java/mate/jdbc/service/DriverServiceImpl.java

```diff
@@ -0,0 +1,40 @@
+package mate.jdbc.service;
+
+import java.util.List;
+import mate.jdbc.dao.DriverDao;
+import mate.jdbc.lib.Inject;
+import mate.jdbc.lib.Service;
+import mate.jdbc.lib.exception.DataProcessingException;
+import mate.jdbc.model.Driver;
+
+@Service
+public class DriverServiceImpl implements DriverService {
+    @Inject
+    private DriverDao driverDao;
+
+    @Override
+    public Driver create(Driver driver) {
+        return driverDao.create(driver);
+    }
+
+    @Override
+    public Driver get(Long id) {
+        return driverDao.get(id).orElseThrow(
+                () -> new DataProcessingException("Could not get driver from DAO by id: " + id));
```

Comment:
```suggestion
                () -> new DataProcessingException("There is no driver with id: " + id + "in db"));
```

We could get driver by id, but there is no driver with such id in the db